packages/test-app-frontend/src/App/ITwinJsApp/ITwinJsApp.tsx

```diff
@@ -280,13 +279,12 @@ class MainFrontstageItemsProvider implements UiItemsProvider {
       content: <ChangedElementsWidget useV2Widget
         feedbackUrl="https://example.com"
         iModelConnection={UiFramework.getIModelConnection()!}
-        manageNamedVersionsSlot={<ManageNamedVersions />}
+        manageNamedVersionsSlot={ManageNamedVersions}
       />,
     }];
```

Comment:
Can you please explain to me why `manageNamedVersionsSlot` needs to be a callback that accepts `IModelConnection` when the user already knows what `IModelConnection` they are going to receive since they already supply it to the same React component? It looks like `<ManageNamedVersions />` could instead accept `IModelConnection` as a prop.

Reply:
This is a direct request from Gintaras and his team. They are the only ones that want to use that prop as far as I know.

Reply:
They wanted it to be simular to this existing option for v1:
`  /**
   * Use this to return a URL to open in a new tab when the user clicks on the 'Manage Named Versions' href in the
   * version compare select dialog. V1 Widget use only.
   */
  getManageNamedVersionsUrl?: (iModelConnection?: IModelConnection) => string;`

Reply:
Even with latest changes, I still don't get why a user could not in the worst case simply do
```tsx
manageNamedVersionsSlot=<ManageNamedVersions onRender={() => computeUrl(getIModelConnection())} />
```